rdr_service/api/participant_summary_api.py

```diff
@@ -56,9 +56,10 @@ def get(self, p_id=None):
             return super(ParticipantSummaryApi, self)._query("participantId")
 
     def _make_query(self, check_invalid=True):
-        constraint_failed, message = self._check_constraints()
-        if constraint_failed:
-            raise BadRequest(f"{message}")
+        if not request.args.get("awardee"):
+            constraint_failed, message = self._check_constraints()
+            if constraint_failed:
+                raise BadRequest(f"{message}")
```

Comment:
Approving since this works for what we need for the hotfix, but I'm worried about how this would fit in with future changes for the ParticipantSummaryApi class. If we need to add more constraints to the query, we'd likely want to add them to the `_check_constraints` method. But if the 'awardee' wasn't part of constraints we'd want to add, then this would need to be modified so we could get the new constraints checked.

It might be better to modify the `_check_constraints` logic to skip the current checks we have if the `awardee` is used.